recipes/litehtml/all/conanfile.py

```diff
@@ -19,10 +19,17 @@ class LitehtmlConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
+        "utf8": [True, False],
+        "with_icu": [True, False],
+        "external_gumbo": [True, False],
+        "external_xxd": [True, False]
```

Comment:
I checked the cmake script, and it looks like xxd is only used at build time.
So I don't think it needs to be an option.
```suggestion
        "with_gumbo": [True, False],
```